bin/build-helper

```diff
@@ -21,6 +25,10 @@ buildImage() {
 }
 
 pushImages() {
+  if [! -f $BASE_DIR/dist/images.txt]
```

Comment:
I am curious about this. It looks like line `32` is going to empty the `images.txt` file. Line `36` seems like it expects to read the values written by `buildImage` but I'm not totally tracking.

Reply:
Yeah, I'm not too clear on the expected functionality here either (eternal bash noob), I just wanted to verify that any time we're interacting with the file it's "safe"